Makefile

```diff
@@ -1,109 +1,54 @@
 # Copyright (c) 2019 the Octant contributors. All Rights Reserved.
 # SPDX-License-Identifier: Apache-2.0
 
```

Comment:
Why do you need a Makefile?

Reply:
it's just mocked out while I test things again CI , it will get removed when I go to mark this ready for review